scylla-cql/src/frame/frame_errors.rs

```diff
@@ -52,3 +54,11 @@ pub enum ParseError {
     #[error(transparent)]
     CqlTypeError(#[from] CqlTypeError),
 }
+
+/// A low level serialization error.
+#[non_exhaustive]
+#[derive(Error, Debug)]
+pub enum LowLevelSerializationError {
+    #[error("Integer conversion out of range: {0}")]
+    TryFromIntError(#[from] std::num::TryFromIntError),
```

Comment:
The error message is: "out of range integral type conversion attempted". I'll leave it as `error(transparent)` then, since we cant provide more context in this case.